src/Components/Members/FormMembers/index.js

```diff
@@ -0,0 +1,291 @@
+            />
+          </div>
+          <div>
+            <TextInput
+              labelName={'Email'}
+              inputName={'email'}
+              changeAction={(e) => onChange(e)}
+              inputType={'text'}
+              text={memberValues.email}
+            />
+          </div>
+          <div>
+            <TextInput
+              labelName={'DNI'}
+              inputName={'dni'}
+              changeAction={(e) => onChange(e)}
+              inputType={'number'}
+              text={memberValues.dni}
+            />
+          </div>
+
+          <div>
+            <TextInput
+              labelName={'Phone'}
+              inputName={'phone'}
+              changeAction={(e) => onChange(e)}
+              inputType={'number'}
+              text={memberValues.phone}
+            />
+          </div>
+          <div>
+            <TextInput
+              labelName={'City'}
+              inputName={'city'}
+              changeAction={(e) => onChange(e)}
+              inputType={'text'}
+              text={memberValues.city}
+            />
+          </div>
+          <div>
+            <TextInput
+              labelName={'PostalCode'}
+              inputName={'postalCode'}
+              changeAction={(e) => onChange(e)}
+              inputType={'number'}
+              text={memberValues.postalCode}
+            />
+          </div>
+          <div className={styles.contDate}>
+            <DatePicker
+              changeAction={(e) => onChange(e)}
+              name={'birthday'}
+              title={'Birthday'}
+              val={memberValues.birthday}
+            />
+          </div>
+          <div className={styles.inputContainer}>
+            <label>Membership</label>
+            <Select
+              changeAction={(e) => onChange(e)}
+              name={'membership'}
+              selectID={''}
+              selectValue={memberValues.membership}
+            >
+              <option value="">Seleccionar</option>
+              <option value="Black Membership">Black Membership</option>
+              <option value="Classic Membership">Classic Membership</option>
+              <option value="Only Classes Membership">Only Classes Membership</option>
+            </Select>
+          </div>
+          <div className={styles.inputContainer}>
+            <label className={styles.label}>Status</label>
+            <Select
+              changeAction={onChange}
+              name={'isActive'}
+              selectID={''}
+              selectValue={memberValues.isActive}
+            >
+              <option value={true}>Active</option>
+              <option value={false}>Inactive</option>
+            </Select>
+          </div>
+        </div>
+        <div className={styles.contButton}>
+          <Button clickAction={submit} text={id ? 'Edit' : 'Add'} type={id ? 'edit' : 'Add'} />
+          <Button clickAction={() => history.goBack()} text={'Cancel'} type={'deleteCancel'} />
```

Comment:
Change the order of the buttons to show on the left the cancel button

Reply:
Also change the colors of the buttons in the form because they have different styles than in the figma.

Reply:
![image](https://github.com/BaSP-m2023/joaco-megarocket-app/assets/94940495/a502db38-8780-4928-b6d6-d8be438abf07)